python/logout_all_users.py

```diff
@@ -4,7 +4,7 @@
 
 
 def main():
-    api_client = client.setup("looker.ini")
+    api_client = client.setup("../looker.ini")
```

Comment:
this should work if you have your `looker.ini` in the root folder

Reply:
Just to be clear, you mean my version with the `../` prefix will work as I expect?

Reply:
yes, I think :-) I see that `looker-sample.ini` is in the root folder so I'd imagine that's where folks should put their `looker.ini` (so it can be used for any language example rather than copying per language folder).

by "root folder" I meant the git root folder (normally `sdk-examples/`):

```bash
 🌊 in ~/Documents/sdk-examples/ on jd/download_tile
› tree
.
├── CODE_OF_CONDUCT.md
├── LICENSE
├── README.md
├── csharp
│   └── README.md
├── java
│   └── README.md
├── kotlin
│   └── README.md
├── looker-sample.ini
├── looker.ini  # <---------------- HERE
├── python
│   ├── Pipfile
│   ├── Pipfile.lock
│   ├── README.md
│   ├── disable_users.py
│   ├── download_tile.py
│   ├── logout_all_users.py
│   └── test_connections.py
.
.
.
```

Reply:
yeah, exactly .. `looker.ini` should be in the **root** of the repository